### PR TITLE
Add Holocene activation timestamps for Sepoila devnets and testnets

### PR DESCRIPTION
**Pull request by sebastianst**
Adds Holocene activation timestamps for Sepolia devnets and testnets.
* ~Sepolia devnets: `Wed Nov 13 15:00:00 UTC 2024`, unix ts `1731510000`, in 2 days~
* ~Sepolia testnets: `Thu Nov 21 15:00:00 UTC 2024`, unix ts `1732201200`, next week Thursday~

_Updated timelines_
* Sepolia devnets (only Base & OP): `Fri Nov 15 15:00:00 UTC 2024`, unix ts `1731682800`, **this Friday**
* Sepolia testnets: `Tue Nov 26 15:00:00 UTC 2024`, unix ts `1732633200`, **Tuesday in 2 weeks**.

This PR also updates all Go versions and CCI images to 1.23

For discussion, see [public discord thread](https://discord.com/channels/1244729134312198194/1305401304381784198).

## Checklist
- [x] I have run `just codegen` to ensure that the chainlist and other generated files are up-to-date and include my chain.

### CODE REVIEW

**Comment by sebastianst** [comment]
Possibly a few more Sepolia chains want to join the Holocene upgrade, namely
```
# cd superchain/configs/sepolia
# rg granite_time
lisk.toml
13:granite_time = 1723478400 # Mon 12 Aug 2024 16:00:00 UTC

superchain.toml
10:granite_time = 1723478400 # Mon Aug 12 16:00:00 UTC 2024

tbn.toml
12:granite_time = 1723545055 #Monday June 12 2024 1:57:35 PM UTC

ethernity.toml
13:granite_time = 1723478400 # Mon 12 Aug 2024 16:00:00 UTC

unichain.toml
13:granite_time = 0 # Thu 1 Jan 1970 00:00:00 UTC

shape.toml
13:granite_time = 1727197200 # Tue 24 Sep 2024 17:00:00 UTC

worldchain.toml
12:granite_time = 1726570800 # Tue 17 Sep 2024 11:00:00 UTC
```

All those chains already activated Grainte at, or after the superchain granite time, so my guess is that they also want to join the Holocene upgrade. Need confirmation @sbvegan.

Note that per Superchain inheritance, the following chains will activate @emilianobonassi 
```
# rg superchain_time
zora.toml
8:superchain_time = 0 # Missing hardfork times are inherited from superchain.toml

base.toml
8:superchain_time = 0 # Missing hardfork times are inherited from superchain.toml

op.toml
7:superchain_time = 0 # Missing hardfork times are inherited from superchain.toml

mode.toml
8:superchain_time = 1703203200 # Fri 22 Dec 2023 00:00:00 UTC
```

**Comment by geoknee** [comment]
@sebastianst if you update the `validation` -> `optimism` dependency on this PR, this should trigger the bot to post a nonzero diff on the `op-node` configs.

File: go.work [hunk 1]
```diff
@@ -1,7 +1,11 @@
-go 1.22
+go 1.22.0
+
+toolchain go1.23.2
```
Comment (geoknee) [inline go.work:3]: I think this line is causing problems with the linter.

**Comment by sbvegan** [comment]
I followed up in the public discord thread and would like a thumbs up from Conduit and the smart contract team to confirm on the Base side.

File: go.work (hunk 1 above)
Comment (sebastianst) [reply go.work]: fixed by upgrading all go versions to 1.23

**Review by sbvegan — APPROVED** [review]
We got the thumbs up from Base and Conduit for their chains

**Review by jelias2 — APPROVED** [review]

**Pull request merged**